src/components/widgets-settings/organizations/organizations.directive.coffee

```diff
@@ -36,21 +36,18 @@ module.controller('SettingOrganizationsCtrl', ($scope, $log, ImpacDashboardsSvc,
         $scope.dashboardOrganizations = config.currentDashboard.data_sources
 
         if w.metadata? && w.metadata.organization_ids?
-          count = 0
           widgetOrgIds = w.metadata.organization_ids
-          currentOrganization = ImpacMainSvc.config.currentOrganization
+          if singleOrgMode()
+            resetSelectedOrgs()
+            if _.map(widgetOrgIds, -> true).length > 1
```

Comment:
why not just `if widgetOrgIds.length > 1`? What's the `_.map` for? If `w.metadata.organization_ids` is defined, it should always be an Array.